### PR TITLE
151 settings

### PR DESCRIPTION
**Pull request by AckslD**
See corresponding pull request for simulaqron with more info at: https://github.com/SoftwareQuTech/SimulaQron/pull/175

### CODE REVIEW

File: cqc/pythonLib.py [hunk 1]
```diff
@@ -233,19 +252,21 @@ def __init__(self, name, socket_address=None, cqcFile=None, appFile=None, appID=
...
-                self.cqcFile = get_cqc_file()
-            else:
-                self.cqcFile = cqcFile
-
-            # Read configuration files for the cqc network
-            if os.path.exists(self.cqcFile):
-                self._cqcNet = networkConfig(self.cqcFile)
+        if socket_address is None or use_classical_communication:
+            if backend is None or backend == "simulaqron":
+                if _simulaqron_major < 3:
```
Comment (LeonWubben) [inline cqc/pythonLib.py:257]: I don't know if this is the best way to go about this. But this should probably be in simulaqron itself as requirement. V2 simulaqron should have cqc <3.0.0

**Review by LeonWubben — COMMENTED** [review]
`make: *** No rule to make target '_verified', needed by 'verify'.  Stop.`
I know it's copied over code, and not part of this pr. Still good to let you know  

I don't know what "make build" does, but if fails as well
```
$ make build
/usr/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
  warnings.warn(msg)
/usr/lib/python3/dist-packages/setuptools/dist.py:294: UserWarning: The version specified ("'3.0.0'") is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
  "details." % self.metadata.version
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'bdist_wheel'
Makefile:28: recipe for target '_build' failed
make: *** [_build] Error 1

```

Should be fine codewise. Can't finish the pr before finishing the simulaqron one though

File: cqc/pythonLib.py (hunk 1 above)
Comment (AckslD) [reply cqc/pythonLib.py:257]: @LeonWubben Why should it be in SimulaQron? Why should simlaqron v2 have cqc<3.0.0, simulaqron v2 works fine with cqc 3 it's the pythonlibrary of cqc 3 which does not work with simulaqron 2.

**Comment by AckslD** [comment]
@LeonWubben What's your version of setuptools? You also need wheel installed.

**Pull request merged**